invenio_rdm_records/config.py

```diff
@@ -329,6 +331,11 @@ def always_valid(identifier):
         client=providers.DataCiteClient("datacite", config_prefix="DATACITE"),
         label=_("DOI"),
     ),
+    providers.OSTIPIDProvider(
```

Comment:
moderate: should we have OSTI added to providers by defautl? Since there is an "enable"/"disable" it is okay. However, it instantiates a provider and there might be credentials errors etc.